x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -155,6 +155,7 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
           {tGridEnabled ? (
             timelinesUi.getTGrid<'embedded'>({
               id,
+              alertConsumers,
```

Comment:
pass alertConsumers for the alert table, but left out of the table in `events_query_tab_body.tsx`